server/test/org/apache/cloudstack/network/lb/CertServiceTest.java

```diff
@@ -440,129 +441,130 @@ public void runUploadSslCertBadkeyPair() throws IOException, IllegalAccessExcept
     public void runUploadSslCertBadkeyAlgo() throws IOException, IllegalAccessException, NoSuchFieldException {
 
         // Reading appropritate files
-        String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed.crt").getFile(),Charset.defaultCharset().name());
-        String keyFile = URLDecoder.decode(getClass().getResource("/certs/dsa_self_signed.key").getFile(),Charset.defaultCharset().name());
+        final String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_self_signed.crt").getFile(),Charset.defaultCharset().name());
+        final String keyFile = URLDecoder.decode(getClass().getResource("/certs/dsa_self_signed.key").getFile(),Charset.defaultCharset().name());
 
-        String cert = readFileToString(new File(certFile));
-        String key = readFileToString(new File(keyFile));
+        final String cert = readFileToString(new File(certFile));
+        final String key = readFileToString(new File(keyFile));
 
-        CertServiceImpl certService = new CertServiceImpl();
+        final CertServiceImpl certService = new CertServiceImpl();
```

Comment:
This line appears in nearly every test method.  Consider converting it to an attribute and initializing it in `setUp`.   Since the `setUp` is run before every test method, the instance will be re-initialized and it will remove some repetition.